docs/app/sidebars.js

```diff
@@ -63,6 +63,21 @@ const sidebars = {
     }, {
         type: 'doc',
         id: 'configuration'
+    }, {
+        type: 'category',
+        label: "IDE Configuration",
+        collapsed: false,
+        items: [{
+            type: 'doc',
+            id: 'ide_configuration/direnv',
+        }, {
+            type: 'doc',
+            id: 'ide_configuration/eclipse',
+        }, {
+            type: 'doc',
+            id: 'ide_configuration/vscode'
+        }]
```

Comment:
should we omit these empty pages?

Reply:
They aren't empty. They look empty in the diff because all I changed was the title. You can see the full file here: https://github.com/jetpack-io/devbox/blob/d3ce7a36afaba679ccba25c1951d28b862030453/docs/app/docs/ide_configuration/eclipse.md

Reply:
they're not empty there are eclipse.md and vscode.md files in `ide_configuration/`